test/metric_value_benchmark/metrics_value_benchmark_test.go

```diff
@@ -64,6 +64,7 @@ func getEc2TestRunners(env *environment.MetaData) []*TestRunner {
 	if ec2TestRunners == nil {
 		factory := &metric.MetricFetcherFactory{Env: env}
 		ec2TestRunners = []*TestRunner{
+			{testRunner: &DiskTestRunner{BaseTestRunner{MetricFetcherFactory: factory}}},
```

Comment:
Using the word "benchmark" to describe these tests is confusing to me. These tests do not compare the metric value against anything.

>verb
evaluate or check (something) by comparison with a standard.
"we are benchmarking our performance against external criteria"

Reply:
@adam-mateen agreed

Reply:
added